tests/integration/defs/perf/disagg/test_disagg.py

```
@@ -0,0 +1,109 @@
+            # Submit job using JobManager
+            success, job_id = JobManager.submit_job(test_config)
+            
+            # Validate submission result
+            assert success, f"Job submission failed: {test_config.test_id}"
+            assert job_id, "Unable to get job ID"
+            
+            # Wait for completion
+            completed = JobManager.wait_for_completion(job_id, 7200)
+            if not completed:
+                JobManager.cancel_job(job_id)
+                assert False, f"Job execution timeout: {job_id}"
```

Ruff (0.14.3)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)